File: core-java-practice/leet-code-codebase/factorial.java
import java.util.*;

public class factorial {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);   // Create Scanner object
        int n = sc.nextInt();                   // Read input number
        int fact = 1;                           // Variable to store factorial
        // Loop to calculate factorial
        for (int i = 1; i <= n; i++) {
            fact = fact * i;
        }

        System.out.println("Factorial = " + fact);

    
    }
}

File: core-csharp-practice/gcr-codebase/csharp-control-flow/level1/Factorial.cs
using System;
class Factorial{
	static void Main(String[]args){
		
		//take number from user
		int number=int.Parse(Console.ReadLine());
		
		//initialize factorial as 1 
		int fact=1;
		
		//assign number to i 
		int i=number;
		while(i>=1){                        //loop to calculate the factorial
			fact=fact*i;			
			i--;
		}
		Console.WriteLine("The Factorial of a number is "+fact);
	}
}